src/etc/platform-intrinsics/powerpc.rs

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Yes, I did not consider the fact there was a pull request open on the master branch when I pushed it...